packages/installer/upgrade.sh

```diff
@@ -218,8 +221,8 @@ while [ "$#" -gt 0 ]; do
         --recovery)
             UPGRADE_RECOVERY=true
             ;;
-        --verify)
-            VERIFY=true
+        --no-verify)
```

Comment:
We should also update docs accordingly as we are changing the default here.. more importantly for derivatives that are not signing images, should set a default `VERIFY=false` in the upgrade channel configuration file, e.g. here https://github.com/rancher-sandbox/cos-toolkit-sample-repo/blob/master/packages/sampleOS/02_upgrades.yaml#L48

Reply:
oh good point, its the second time I change this around so docs need updating.

Will send also a patch for the derivatives

Reply:
done!

Tests still need rework thought, that can come afterwards in a different PR to rework the whole upgrade testing part.